src/Codemirror.js

```diff
@@ -1,26 +1,28 @@
 const React = require('react');
 const ReactDOM = require('react-dom');
 const findDOMNode = ReactDOM.findDOMNode;
```

Comment:
🤔 Wonder if we should switch this to callback refs to get away from using `findDOMNode`.